### PR TITLE
Overhaul Cloud-Init ipconfig v2

### PR DESCRIPTION
**Pull request by sebastian-de**
Discussion in issue #214 
Supersedes #215

### CODE REVIEW

**Comment by mleone87** [comment]
Hey @sebastian-de this should be good! I would only add a check to not exceed 16 ipconfig elements, or at least that we cannot have ipconfig16 and upper, this would give an api error for sure

**Comment by sebastian-de** [comment]
Hey @mleone87 thanks for the review! I'm away for the rest of the week - I'll add the required change next Monday/Tuesday.

File: proxmox/config_qemu.go [hunk 1]
```diff
@@ -1337,6 +1242,23 @@ func (c ConfigQemu) CreateQemuNetworksParams(vmID int, params map[string]interfa
 	return nil
 }
 
+// Create parameters for each Cloud-Init ipconfig entry.
+func (c ConfigQemu) CreateIpconfigParams(vmID int, params map[string]interface{}) error {
+
+	for ID, config := range c.Ipconfig {
+		if ID > 15 {
```
Comment (sebastian-de) [inline proxmox/config_qemu.go:1249]: Check added!

**Comment by sebastian-de** [comment]
@mleone87 gentle ping - I added the check, is there anything else that needs to be done to get this merged?

**Comment by mleone87** [comment]
> @mleone87 gentle ping - I added the check, is there anything else that needs to be done to get this merged?

so sorry that I completely miss the commit!

**Pull request merged**